src/tests/utils/setupTests.js

```diff
@@ -0,0 +1,2 @@
+import "@testing-library/jest-dom";
+import "@testing-library/jest-dom/extend-expect";
```

Comment:
These imports are needed to use some of the jest-dom matchers i.e. `toBeInTheDocument`. Rather than importing them again and again, this one setup file will import for all